src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/JtsAdapter.java

```diff
@@ -512,14 +499,9 @@ public static TileGeomResult createTileGeom(List<Geometry> g,
         final int repeatEndCoordCount = countCoordRepeatReverse(geomCoords);
         final int minExpGeomCoords = geomCoords.length - repeatEndCoordCount;
 
-        // Guard/Optimization: Not enough geometry coordinates for a line
-        if(minExpGeomCoords < 2) {
-            Collections.emptyList();
```

Comment:
I think instead of removing emptyList() call, can we just define our own 'emptyList()' convenience function in a utility class that returns a unmodifiable static list value that is empty?

Reply:
as above - no assignment

Reply:
I think this is meant to return Collections.emptyList(), but the return statement is missing. Could be a bug discovery rather than dead code discovery.